ecommerce_api_client/tests/test_auth.py

```diff
@@ -18,32 +24,57 @@ def setUp(self):
         self.signing_key = 'edx'
         httpretty.register_uri(httpretty.GET, self.url)
 
-    def assert_expected_token_value(self, tracking_context=None):
+    def assert_expected_token_value(self, tracking_context=None, issuer=None, expires_in=30):
```

Comment:
`expires_in` should default to `None` so that consuming tests are not required to mock `datetime_now` when using this method.